python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2

```diff
@@ -99,6 +99,9 @@ dot1x
    radius av-pair framed-mtu {{ dot1x.radius_av_pair.framed_mtu }}
 {%             endif %}
 {%         endif %}
+{%         if dot1x.mac_based_auth_radius.delimiter is arista.avd.defined and dot1x.mac_based_auth_radius.mac_string_letter_case is arista.avd.defined %}
```

Comment:
The CLI is very different than the variable. I think you should change the model to be `av_pair_user_name_delimiter` since there could be other options coming later.

Reply:
same for case. Maybe find a better variable name there too. `av_pair_user_name_case` maybe? (please check wording on EOS)

Reply:
I updated the schema as below:
radius_av_pair_username_format:
  delimiter:
  mac_string_case:
  
  EOS shows below description for lowercase/uppercase setting.
  
  s1-leaf2(config-dot1x)#mac-based-auth radius av-pair user-name delimiter colon  ?
  lowercase  MAC address string in lowercase
  uppercase  MAC address string in uppercase